### PR TITLE
Beautify migration page

### PR DESCRIPTION
**Pull request by matthias-ronge**
- improve layout
- improve texts
- show tasks with white spaces separated by comma

beautiy2

### CODE REVIEW

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 1]
```diff
@@ -541,7 +541,9 @@ metsRightsOwnerMail=METS Rechteinhaber Kontakt
 metsRightsOwnerSite=METS Rechteinhaber URL
 metsfilegroups=Unterordner und METS Dateigruppen
 migrateMetadata=Metadaten konvertieren
-migrateProjects=Migrieren
+migrateMetadataHint=Die Konvertierung wurde im Taskmanager gestartet.
+migrateMetadataOfSelectedProjects=Metadaten dieser Projekte konvertieren
+migrateProjects=Nach Vorg\u00E4ngen suchen
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/resources/messages/messages_de.properties]: This change also changes the name of the tab
migrationTab
I think "Migration" would fit better here

**Comment by Kathrin-Huber** [comment]
To make the Seleniumtest work with the comma-separated workflow name, you need to rebase it on #2958

**Comment by matthias-ronge** [comment]
@Kathrin-Huber I had a rebase on the current master, but it still seems to hang somehow. Do you know why?

**Comment by Kathrin-Huber** [comment]
The branch was still failing :-(
Wait until the PR is successfully checked by travis, then rebase again, sorry for the inconvenience

**Comment by Kathrin-Huber** [comment]
You can rebase on master now

**Pull request merged**